pytest_html_merger/main.py

```diff
@@ -1,21 +1,22 @@
-import json
```

Comment:
Please add only necesssary imports. If change of order is introduced, please submit in a new pull request